assets/js/script.js

```diff
@@ -6,12 +6,15 @@ document.addEventListener("DOMContentLoaded", function () {
 
     for (let i = 0; i < response.length; i++) {
```

Comment:
I suggest you to use `forEach` instead. this would make your code more declarative.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach#converting_a_for_loop_to_foreach